### DISCUSSION TITLE
/nice-article/itmind/navicatmacyjpx.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#78] [Announcements]
# /nice-article/itmind/navicatmacyjpx.html

Navicat 16 macOS 永久破解版本，不需要注册机，绿色安装

https://tobebetterjavaer.com/nice-article/itmind/navicatmacyjpx.html

**Comment by grantcguo** [comment]
为什么打开的时候提示文件被破坏了
